planning/behavior_path_planner/src/scene_module/lane_change/interface.cpp

```diff
@@ -474,33 +494,6 @@ LaneChangeBTModule::LaneChangeBTModule(
 {
 }
 
-ModuleStatus LaneChangeBTModule::updateState()
-{
-  if (!module_type_->isValidPath()) {
-    current_state_ = ModuleStatus::FAILURE;
-    return current_state_;
-  }
-
-  if (module_type_->isAbortState()) {
-    current_state_ = ModuleStatus::RUNNING;
-    return current_state_;
-  }
-
-  if (module_type_->isCancelConditionSatisfied()) {
-    current_state_ =
-      module_type_->isCancelEnabled() ? ModuleStatus::FAILURE : ModuleStatus::RUNNING;
-    return current_state_;
-  }
-
-  if (module_type_->hasFinishedLaneChange()) {
-    current_state_ = ModuleStatus::SUCCESS;
-    return current_state_;
-  }
-
-  current_state_ = ModuleStatus::RUNNING;
-  return current_state_;
-}
-
```

Comment:
Recently `updateState` is separated between `LaneChangeInterface::updateState()` and `LaneChangeBTInterface`. Therefore the `updateState` of `LaneChangeBTInterface` also needs to be included/updated.